### PR TITLE
Basic audio recording

### PR DESCRIPTION
**Pull request by Xelowak**
Implement audio recording checklist :
- [x] Audio recording
- [x] Audio playing
- [x] Basic UI
- [x] Testing UI
- [ ] Testing audio
- Used MediaRecorder to record audio input from device microphone
- Audio file currently saved in an external cache for visibility
- Used MediaPlayer to play the recorded audio
- Added an UI with buttons to start/stop recording and playing
- Requested for user permission to use the microphone (tests of permissions don't pass yet)
- Impossible to test audio recording with the emulator, might need an Android smartphone

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -2,14 +2,16 @@
...
+
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.Blindly">
-        <activity android:name=".MainActivity">
+        <activity android:name=".recording.RecordingActivity">
```
Comment (r-ismer) [inline app/src/main/AndroidManifest.xml:14]: this activity is not the main activity be carrefull

**Review by r-ismer — APPROVED** [review]
ok but be carrefull for main activity

**Review by Lollipoke — COMMENTED** [review]
Seems good to me, apart from the main activity comment from Richard

**Review by Lollipoke — APPROVED** [review]

File: app/src/main/java/ch/epfl/sdp/blindly/MainActivity.kt [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
     }
+
+    fun launchRecording(view: View) {
+        val intent = Intent(this, RecordingActivity::class.java)
+        startActivity(intent)
+    }
```
Comment (Xelowak) [inline app/src/main/java/ch/epfl/sdp/blindly/MainActivity.kt:18]: Useless atm, might delete

**Pull request merged**